### DISCUSSION TITLE
4 day trip->experiences

### DISCUSSION THREAD

**Discussion by Leeb64** [devemux86/cruiser#536] [Show and tell]
I was traveling a bit over the long weekend (4 days).
Planning and navigating went smoothly!!!!
However, one has to complain about the turn-by-turn announcements.
I don't need an announcement when I'm on a main road and pass a dirt road... something like "turn slightly left (or right)".
Announcements like this come up very often!
As long as you stay on a certain street, you don't need such announcements(Its  missleading). These announcements sometimes come when driving past rest areas.
Next is the ride in tunnels.
If the GPS signal is lost in tunnels, CRUISER will not follow the road through the tunnel.
I don't know how other navigation manufacturers manage it - but there seems to be a semi-working solution.
My ancient Becker navigation system followed the road at the speed it was currently driving.
It wasn't perfect, but it worked.
Otherwise everything worked fine.

**Comment by devemux86** [comment]
@Leeb64 thanks for sharing your experience! :+1: 

> turn-by-turn announcements

Turn instructions are generated by the routers. In your case, BRouter?

Cruiser is already processing them, removing some unnecessary "Continue" instructions.

> ride in tunnels

This is indeed a complicated topic for all navigators. It is already in my list:

- https://github.com/devemux86/cruiser/discussions/535

**Reply by Leeb64** [reply]
BRouter !

**Comment by DIrgi** [comment]
Hi, as devemux mentioned it's and old issue topic. 
In former times, all systems act like this. I assume, newer systems does a kind of a simulation - because GPS is still missing. Depending on the parameters they have, last speed before entering the tunnel, allowed max speed in the tunnel (given by map or road signs) and maybe connected to vehicle electronics (Odometer), they let you more or less precise navigate you through a tunnel.
You can check how accurate the system is, when you leave the tunnel and how closer your pointer is then in reality.
Since the origin is always still based on a satellite signal there are no more exact navigations available.
So finally it's not so easy to find the most reliable setup for a programmer but I'm confidet, devemux will find a solution when it's "on his list"

**Reply by devemux86** [reply]
It is still in my list, too much work on Cruiser lately. :slightly_smiling_face: 

We have a topic here:
- https://github.com/devemux86/cruiser/discussions/535
